### PR TITLE
Create 2020_12_chng_partnership.md

### PR DESCRIPTION
**Pull request by christymelucci**
New news post: CHNG partnership

### CODE REVIEW

File: content/about/news/headless/2020_12_chng_partnership [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
```
Comment (sgratzl) [inline content/about/news/headless/2020_12_chng_partnership:1]: the file has to end with the `.md` extension

File: content/about/news/headless/2020_12_chng_partnership [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+date: 2020-12-01
+title: COVIDcast enhanced with Change Healthcare COVID-19 de-identified claims data
+category: update
+---
+
+[Change Healthcare and Carnegie Mellon University’s Delphi Research Group announce the launch of Delphi's enhanced COVIDcast real-time COVID-19 indicators] (https://newsroom.changehealthcare.com/press-releases/carnegie-mellon-university-s-delphi-research-group-enhances-covi)
```
Comment (sgratzl) [inline content/about/news/headless/2020_12_chng_partnership:7]:
```suggestion
[Change Healthcare and Carnegie Mellon University’s Delphi Research Group announce the launch of Delphi's enhanced COVIDcast real-time COVID-19 indicators](https://newsroom.changehealthcare.com/press-releases/carnegie-mellon-university-s-delphi-research-group-enhances-covi)
```

removed the whitespace for the link

**Comment by sgratzl** [comment]
-> https://deploy-preview-369--cmu-delphi-main.netlify.app/about/news/

**Review by sgratzl — APPROVED** [review]
👍

**Review by krivard — APPROVED** [review]
👍

**Pull request merged**